src/output-json-http.c

```diff
@@ -350,6 +352,68 @@ static void JsonHttpLogJSONExtended(json_t *js, htp_tx_t *tx)
     json_object_set_new(js, "length", json_integer(tx->response_message_len));
 }
 
+static void BodyPrintableBufferCopy(json_t *js, HtpBody *body, const char *key)
```

Comment:
http body was copied in a buffer the first time I have developed the function, so 'copy' was for this reason.